conf/dev.conf

```diff
@@ -3,3 +3,13 @@ include "portal.application.conf"
 
 # Development configuration
 # <Nothing>
+persistence {
```

Comment:
This doesn't address running `start` locally, as the metrics repo I believe does. If we're going to punt on the real configuration then you don't need the dev config (because that's what cluster.conf is).

Reply:
Removed dev.conf.